src/psyclone/parse/algorithm.py

```diff
@@ -50,18 +51,24 @@
     Section_Subscript_List, Name, Real_Literal_Constant, \
     Int_Literal_Constant, Function_Reference, Level_2_Unary_Expr, \
     Add_Operand, Parenthesis, Structure_Constructor, Component_Spec_List, \
-    Proc_Component_Ref
+    Proc_Component_Ref, Kind_Selector, Type_Declaration_Stmt, \
+    Declaration_Type_Spec, Entity_Decl, Intrinsic_Type_Spec, \
+    Data_Component_Def_Stmt, Component_Decl
 # pylint: enable=no-name-in-module
 
 from psyclone.configuration import Config
 from psyclone.parse.utils import check_api, check_line_length, ParseError, \
     parse_fp2
 from psyclone.errors import InternalError
+from psyclone.parse.kernel import BuiltInKernelTypeFactory, get_kernel_ast, \
```

Comment:
In line with our new 'imports must be in alpha order' rule, please could you put this line before the `from psyclone.parse.utils import...` line.

Reply:
I must have missed that new rule - unless it is to do with your latest pylint? I also moved errors as presumably that should be after configuration? What about capitals. do they go after all lower case?

Reply:
It was a rule that @hiker imagined we had and I thought was a good idea so now it is a rule :-) I think I would just ignore case.